internal/cli/cmd/connectivity.go

```diff
@@ -132,6 +132,7 @@ func newCmdConnectivityTest() *cobra.Command {
 	cmd.Flags().StringVar(&params.PerformanceImage, "performance-image", defaults.ConnectivityPerformanceImage, "Image path to use for performance")
 	cmd.Flags().StringVar(&params.JSONMockImage, "json-mock-image", defaults.ConnectivityCheckJSONMockImage, "Image path to use for json mock")
 	cmd.Flags().StringVar(&params.DNSTestServerImage, "dns-test-server-image", defaults.ConnectivityDNSTestServerImage, "Image path to use for CoreDNS")
+	cmd.Flags().BoolVar(&params.Datapath, "datapath", false, "Run datapath conformance tests")
```

Comment:
I wonder whether we should mark this as hidden for now, given the following in the commit message:

> For now, the test case is protected by the flag "--datapath". In the
future we might drop the flag. I.e., the test cases will be part of the
regular connectivity suite.

Reply:
SGTM!